File: apps/ios/Pebbles/Theme/Spacing.swift
import CoreGraphics

/// Six-step spacing scale rooted on the 17pt iOS body baseline.
/// `lg` equals the body font size (17), `xxl` equals `lg * 2` (34).
/// Use these constants in place of literal pt values for paddings,
/// stack spacings, and corner radii so visual rhythm stays consistent
/// across screens.
enum Spacing {
    static let xs:  CGFloat = 3
    static let sm:  CGFloat = 10
    static let md:  CGFloat = 13
    static let lg:  CGFloat = 17    // root, == iOS body font size
    static let xl:  CGFloat = 22
    static let xxl: CGFloat = 34    // == lg * 2
}

File: apps/android/app/src/main/kotlin/app/pbbls/android/theme/Spacing.kt
package app.pbbls.android.theme

import androidx.compose.ui.unit.Dp
import androidx.compose.ui.unit.dp

/**
 * Six-step spacing scale rooted on the 17dp body baseline. `lg` equals the
 * body font size (17), `xxl` equals `lg * 2` (34). Use these in place of
 * literal dp values for paddings, stack spacings, and corner radii so visual
 * rhythm stays consistent across screens. Mirrors
 * `apps/ios/Pebbles/Theme/Spacing.swift`.
 */
object Spacing {
    val xs: Dp = 3.dp
    val sm: Dp = 10.dp
    val md: Dp = 13.dp
    val lg: Dp = 17.dp // root, == body font size
    val xl: Dp = 22.dp
    val xxl: Dp = 34.dp // == lg * 2
}
